src/mono/netcore/System.Private.CoreLib/src/System/Threading/ThreadPool.Mono.cs

```diff
@@ -32,6 +33,7 @@ public static bool BindHandle(IntPtr osHandle)
             throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported); // Replaced by ThreadPoolBoundHandle.BindHandle
         }
 
+        [UnsupportedOSPlatform("browser")]
```

Comment:
Since it looks like ThreadPool.BindHandle is not supported on any of mono, I wonder if it would be better to add support for `UnsupportedOSPlatform("mono")` or to list `UnsupportedOSPlatform` attribute for each mono platform for now. I believe having just `UnsupportedOSPlatform("browser")` would imply that this API is supported on the other mono platforms.

Reply:
I think this API is Windows specific (cc @jkotas for confirmation) and we could mark it that way. Using `UnsupportedOSPlatform("mono")` combination does not make sense.

Reply:
Yes, `BindHandle` is Windows-specific. CoreCLR implementation throws PNSE on Unix.

Reply:
@mdh1418 could you turn the attribute to `[SupportedOSPlatform("windows")]`